src/pilet-debug.test.ts

```diff
@@ -40,4 +41,200 @@ runTests('pilet-debug', ({ test, setup }) => {
+    'can produce a debug build with none schema',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --schema none`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      const res = await axios.get(`http://localhost:${port}/$pilet-api`);
+      const pilets = res.data;
+
+      expect(pilets).toEqual({
+        name: expect.anything(),
+        version: expect.anything(),
+        link: expect.anything(),
+        spec: 'v0',
+        hash: expect.anything(),
+        noCache: expect.anything(),
+      });
+
+      await expect(page).not.toHaveSelectorCount('.pi-tile', 1);
+
+      await expect(page).not.toMatchText('.pi-tile', 'Welcome to Piral!');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-an-external-feed',
+    'can produce a debug build with an external feed',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --feed "https://feed.piral.cloud/api/v1/pilet/sample"`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      await expect(page).toHaveSelectorCount('.mario-tile', 1);
+      await expect(page).toHaveSelectorCount('.spaceshoot-tile', 1);
+      await expect(page).toHaveSelectorCount('.pi-tile', 1);
+      await expect(page).toMatchText('.pi-tile', 'Welcome to Piral!');
+      await expect(page).toMatchText('.mario-tile', 'Mario5');
+      await expect(page).toMatchText('.spaceshoot-tile', 'Spaceshoot');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-an-external-feed',
+    'can produce a debug build with an external feed',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --feed "https://feed.piral.cloud/api/v1/pilet/sample"`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      await expect(page).toHaveSelectorCount('.mario-tile', 1);
+      await expect(page).toHaveSelectorCount('.spaceshoot-tile', 1);
+      await expect(page).toMatchText('.mario-tile', 'Mario5');
+      await expect(page).toMatchText('.spaceshoot-tile', 'Spaceshoot');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-different-app-shell',
+    'can produce a debug build with different app shell',
+    ['debug.pilet'],
+    async (ctx) => {
+      await ctx.run(`npm i netflix-piral`);
+
+      await ctx.setFiles({
+        'src/index.tsx'(content: string) {
+          content = `import * as React from 'react';
+import { PiletApi } from 'sample-piral';
+          
+export function setup(app: PiletApi) {
+  app.showNotification('Hello from Piral!', {
```

Comment:
That's a bad setup as it leaves us with no option to verify that the instance is working. I'd recommend adding a menu item.

Reply:
Alternatively, register a page called `browse`. It would / should be shown then when you run it.